README.md

```diff
@@ -30,6 +30,7 @@ For instance, please use the "New Package Request" issue type to [file your requ
 
 For quick questions or for just hanging out with the community please use
 * Our matrix chat (via element.io): [https://app.element.io/#/room/#flatcar:matrix.org](https://app.element.io/#/room/#flatcar:matrix.org)
+* Our IRC channel (Libera.chat): #[flatcar](ircs://irc.libera.chat:6697/#flatcar)
```

Comment:
Thanks for the update.
As the matrix channel is primary, can we word this like 
```suggestion
* A bridged IRC channel (Libera.chat): #[flatcar](ircs://irc.libera.chat:6697/#flatcar) channel is also available
```

Reply:
or something similar. When we moved off freenode, we had intentionally _not_ mentioned a replacement IRC channel :-)